.azure-pipelines-canary.yml

```diff
@@ -43,9 +43,4 @@ stages:
       androidKeyStoreFile: $(InternalKeystore)
       androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
       appCenterServiceConnectionName: $(AppCenterCanaryServiceConnection)
-      appCenterDistributionGroup: $(AppCenterCanaryDistributionGroup)
-
-  - template: build/stage-release-appstore.yml
```

Comment:
So we're not supposed to use TestFlight for the canaries?

Reply:
We can but in my experience App center is better for canaries, the builds are not expired after 90 days like in TestFlight.

Reply:
But don't you have to also create a new certificate and parametrize the pipelines to support 2 iOS certificates?

Reply:
We do not need to create a new certificate for Canaries, it would be on preference, do you want to mix the builds for canaries and non canaries. If we seperate them, yes we need 2 iOS certificates.